enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -126,7 +126,7 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
         LinkageCheckReport linkageReport = linkageChecker.findLinkageErrors();
         int errorCount =
             linkageReport.getJarLinkageReports().stream()
-                .mapToInt(JarLinkageReport::getCauseToSourceClassesSize)
```

Comment:
This should not be included as part of this change. `getCauseToSourceClassesSize` is the correct method.